plugins/home/src/components/HomepageCompositionRoot.tsx

```diff
@@ -16,18 +16,12 @@
 
 import React, { ReactNode } from 'react';
 import { useOutlet } from 'react-router';
-import { Content, Header, Page } from '@backstage/core-components';
 
 export const HomepageCompositionRoot = (props: {
   title?: string;
   children?: ReactNode;
 }) => {
   const outlet = useOutlet();
   const children = props.children ?? outlet;
-  return (
-    <Page themeId="home">
-      <Header title={props.title ?? 'Home'} />
-      <Content>{children}</Content>
-    </Page>
-  );
+  return <div>{children}</div>;
```

Comment:
We should really consider this a breaking change, since it'll break the layout of existing users. Meaning we wanna change the changeset level to `minor` and perhaps give a little bit of guidance for how to migrate.

Reply:
Only other comment is that if the `<div>` is here for type satisfying purposes, you could probably also just use empty fragments `<>{children}</>`, unless the `<div>` is serving a layout purpose.

Reply:
I'll switch this to the empty fragments, the `<div>` was just came to mind to satisfy the type, thanks for pointing this out

Reply:
I've pushed up a change for the empty fragment